### PR TITLE
feat: ASC-22100 implement empty newsfeed component

### PR DESCRIPTION
**Pull request by NaingAmity**

### CODE REVIEW

File: src/v4/enum/enumUIKitID.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export enum ElementID {
   illustration = 'illustration',
   title = 'title',
   description = 'description',
-  explore_communities_button = 'explore_communities_button',
+  explore_communities_button = 'explore_communitties_button',
```
Comment (frankent) [inline src/v4/enum/enumUIKitID.ts:54]:
```suggestion
  explore_communities_button = 'explore_communities_button',
```

File: src/v4/PublicApi/Components/AmityEmptyNewsFeedComponent/Elements/ExploreCommunityButton.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  const icon = useConfigImageUri({
+    configKey: 'icon',
+    configPath: {
+      page: PageID.social_home_page,
+      component: ComponentID.empty_newsfeed,
+      element: ElementID.explore_communities_button,
+    },
+  });
+
+  const onPressExploreCommunity = useCallback(() => {}, []);
```
Comment (frankent) [inline src/v4/PublicApi/Components/AmityEmptyNewsFeedComponent/Elements/ExploreCommunityButton.tsx:28]: Should we remove this ?

**Review by frankent — COMMENTED** [review]
@NaingAmity Please check all my comments

File: src/v4/enum/enumUIKitID.ts (hunk 1 above)
Comment (NaingAmity) [reply src/v4/enum/enumUIKitID.ts:54]: That wrong type string `explore_communitties_button` is from config json in tech spec. I am not sure should I correct spelling or not. That's why I use the wrong typo xD

Should I change ?

File: src/v4/PublicApi/Components/AmityEmptyNewsFeedComponent/Elements/ExploreCommunityButton.tsx (hunk 2 above)
Comment (NaingAmity) [reply src/v4/PublicApi/Components/AmityEmptyNewsFeedComponent/Elements/ExploreCommunityButton.tsx:28]: At that time, PDM and Stake holders does not provide the action of that function. I got the answer last night, I will implement it

File: src/v4/enum/enumUIKitID.ts (hunk 1 above)
Comment (NaingAmity) [reply src/v4/enum/enumUIKitID.ts]: Updated xD

File: src/v4/PublicApi/Components/AmityEmptyNewsFeedComponent/Elements/ExploreCommunityButton.tsx (hunk 2 above)
Comment (NaingAmity) [reply src/v4/PublicApi/Components/AmityEmptyNewsFeedComponent/Elements/ExploreCommunityButton.tsx]: Updated !

File: src/v4/enum/enumUIKitID.ts (hunk 1 above)
Comment (frankent) [reply src/v4/enum/enumUIKitID.ts]: > That wrong type string `explore_communitties_button` is from config json in tech spec. I am not sure should I correct spelling or not. That's why I use the wrong typo xD
> 
> Should I change ?

I like to say YES, and please ping others about this thing as well

**Review by frankent — APPROVED** [review]
LGTM, Please ping the others (owner who create this config) about the spelling for updating as well

File: src/v4/enum/enumUIKitID.ts (hunk 1 above)
Comment (NaingAmity) [reply src/v4/enum/enumUIKitID.ts]: Yes. already changed and will let them know

**Pull request merged**